source/scripts/script.js

```diff
@@ -124,23 +137,28 @@ function createRecipeCards() {
 
 function bindRecipeCard(recipeCard, pageName) {
     recipeCard.addEventListener('click', e => {
-        if (e.path[0].nodeName == "A") return;
-        router.navigate(pageName);
+      if (e.path[0].nodeName == 'A') return;
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group23/cse110-fa21-group23/pullRequest?prid=8436588)
```suggestion
        if (e.path[0].nodeName == "A") return;
```